src/dmd/globals.d

```diff
@@ -561,6 +561,8 @@ struct Loc
     const(char)* filename; // either absolute or relative to cwd
     uint linnum;
     uint charnum;
+    version (DMDLIB)
+        uint offset;
```

Comment:
`offset` is heavily used in DMD, making it un-greppable. Instead, use fileOffset or whatever it is an offset from.